90.md

```diff
@@ -90,6 +121,27 @@ Service providers publish job results, providing the output of the job result. T
 * `amount`: millisats that the Service Provider is requesting to be paid. An optional third value can be a bolt11 invoice.
 * `i`: The original input(s) specified in the request.
 
+## Encrypted Output
+
+If the request has encrypted params, then output should be encrypted and placed in  `content` field with `p` tag. If the output is encrypted, then avoid including `i` tag with input-data as clear text. 
```

Comment:
> with `p` tag.

What does this mean? Does it mean that the customer's pubkey should be `p` tagged? This is always the case so there's nothing special about it being encrypted or not; if that's what's meant here we should remove this since it makes it more confusing.